src/getDefaultHostingSite.ts

```diff
@@ -0,0 +1,19 @@
+import * as logger from "./logger";
+import { getFirebaseProject } from "./management/projects";
+import * as _ from "lodash";
+
+/**
+ * Tries to determine the default hosting site for a project, else falls back to projectId.
+ * @param {Object} options The command-line options object
+ * @returns {Promise<String>} The hosting site ID
+ */
+export async function getDefaultHostingSite(options: any): Promise<string> {
+  const project = await getFirebaseProject(options.project);
+  if (!_.has(project, "resources.hostingSite")) {
+    logger.debug(
+      `No default hostingSite found for project: ${options.project}. Using projectId as hosting site name.`
+    );
+    return options.project;
+  }
+  return _.get(project, "resources.realtimeDatabaseInstance");
```

Comment:
1. Should this be returning `hostingSite`?
2. Maybe this function can drop lodash completely and save off the hosting site once:

```typescript
  const site = project.resources?.hostingSite;
  if (!site) {
    logger.debug(
      `No default hostingSite found for project: ${options.project}. Using projectId as hosting site name.`
    );
    return options.project;
  }
  return site;
```